### PR TITLE
Et staging customisations

### PR DESCRIPTION
**Pull request by Humairaa127**
Added form customisations in et-v1
Updated index.js to reference et-v1
Updated permissions to include et.ts

### CODE REVIEW

**Review by murilovmachado — APPROVED** [review]
Looks good to me!

@GPaoloni can you review the Insights related definitions? I'm not super familiar with that. Also, when this is merged and deployed, we'll need to update these `serviceConfiguration` properties: 
```
"permissionConfig": "za-v1",
"definitionVersion": "za-v1",
```
to:
```
"permissionConfig": "et",
"definitionVersion": "et-v1",
```

And a last question @GPaoloni : I see that the current `permissionConfig` is set to `za-v1`, which doesn't exist. The acceptable values are `type PermissionConfig = 'zm' | 'za' | 'et';`. Ethipia may have issues currently related to permissions.

File: plugin-hrm-form/src/formDefinitions/et-v1/HelplineInformation.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "label": "Helpline",
+  "helplines": [
+    {
+      "label": "Adama ChildLine",
+      "value": "ChildLine Ethiopia (ET)",
```
Comment (murilovmachado) [inline plugin-hrm-form/src/formDefinitions/et-v1/HelplineInformation.json:6]: Here the value I believe should be `Adama Child Helpline` as well.
@GPaoloni can you double-check that?

File: plugin-hrm-form/src/formDefinitions/et-v1/tabbedForms/IssueCategorizationTab.json [hunk 2]
```diff
@@ -0,0 +1,133 @@
+{
+  "ChildLine Ethiopia (ET)": {
```
Comment (murilovmachado) [inline plugin-hrm-form/src/formDefinitions/et-v1/tabbedForms/IssueCategorizationTab.json:2]: Here too, I suspect the helpline is set to be `Adama ChildLine` on Ethiopia Twilio Console.
Or should we update the Twilio Console to be `ChildLine Ethiopia (ET)`?

File: plugin-hrm-form/src/formDefinitions/et-v1/HelplineInformation.json (hunk 1 above)
Comment (Humairaa127) [reply plugin-hrm-form/src/formDefinitions/et-v1/HelplineInformation.json]: change made

File: plugin-hrm-form/src/formDefinitions/et-v1/tabbedForms/IssueCategorizationTab.json (hunk 2 above)
Comment (Humairaa127) [reply plugin-hrm-form/src/formDefinitions/et-v1/tabbedForms/IssueCategorizationTab.json]: change made

**Pull request merged**